doc/translations/ar.po

```diff
@@ -1629,7 +1629,7 @@ msgstr ""
 #: doc/classes/@GlobalScope.xml
 msgid ""
 "The [JavaScript] singleton.\n"
-"[b]Note:[/b] Only implemented on HTML5."
+"[b]Note:[/b] Only implemented on Web."
```

Comment:
It's best to leave translations unchanged, they will be synced later (and currently they are actually synced with `3.x` so they shouldn't be changed here at all).